rs/registry/canister/src/mutations/common.rs

```diff
@@ -272,74 +142,4 @@ pub(crate) mod test {
         assert!(check_ipv6_format("0:0:0:0:0:0:0:0"));
         assert!(check_ipv6_format("123:221:4567:323:4123:2111:7:7"));
     }
-
-    #[test]
-    fn test_is_valid_ipv4_address() {
```

Comment:
note for reviewer: this test doesn't make sense since it is testing std::net::Ipv4Addr::from_str